src/rmkit/ui/dropdown.cpy

```diff
@@ -121,7 +127,7 @@ namespace ui:
         self.scene = ui::make_scene()
         y_off := self.option_y
         if dir == DIRECTION::DOWN:
-          y_off = y - self.option_y
+          y_off = y + self.option_y
```

Comment:
I also came across this while testing puzzles -- I think this was a bug? If I had a dropdown with DIRECTION::DOWN and called `my_dropdown->set_option_offset(0, my_dropdown->h)` the scene ended up offset above the dropdown, which made the first option draw offscreen.

Reply:
yes, i think it's likely a bug, but i'd have to play with it to understand. looks like only dithering demo is using DIRECTION::DOWN atm